dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -296,6 +296,7 @@ private static void setMetadataRelatedIdentifiers(DepositMetadata depositMetadat
      * @param workflow    workflow for which DOI is registered
      */
     private static void setMetadataCreator(DepositMetadata depositMetadata, Workflow workflow) {
+        // TODO: remove usage of getAuthor() below in https://github.com/dockstore/dockstore/issues/5437
```

Comment:
Ah, does this tie into my other comment about whether we still need `getAuthor()`? I guess when I update this to add the list of authors, we can remove `Entry.getAuthor()`?

Reply:
Given the likely date of the release, think deprecate in 1.14 and then remove in 1.15 sounds safe